src/main/java/com/mabl/integration/jenkins/MablStepDeploymentRunner.java

```diff
@@ -118,14 +122,14 @@ public Boolean call() {
         }
     }
 
-    private void execute() throws MablSystemError, MablPlanExecutionFailure {
+    private void execute() throws MablSystemException, MablPlanExecutionFailure {
         // TODO descriptive error messages on 401/403
         // TODO retry on 50x errors (proxy, redeploy)
         outputStream.printf("mabl is creating a deployment event:%n  environment_id: [%s]%n  application_id: [%s]%n  labels: [%s]  branch: [%s]%n",
                 environmentId == null ? "empty" : environmentId,
                 applicationId == null ? "empty" : applicationId,
-                labels == null ? "empty" : StringUtils.join(labels, ", "),
-                mablBranch == null ? "master" : mablBranch
+                labels == null ? "empty" : labels,
+                mablBranch == null ? "main" : mablBranch
```

Comment:
I do not think we can change that until we change default in mabl branching code itself. Right now default branch in mabl is master. I am referring to creating branches and merging in trainer and app.

Reply:
It is just a print statement, nothing else, but I will revert it back.